src/cli.ts

```diff
@@ -74,7 +74,10 @@ const mainCommand = defineCommand({
     });
 
     const _from = r.name || r.url;
-    const _to = relative(process.cwd(), r.dir);
+    const _to =
+      process.cwd() === r.dir
+        ? "the current directory"
```

Comment:
Maybe show the full path in this situation to avoid confusion? ideas welcome

Reply:
Maybe we just display the entire path all the time?

Reply:
Using relative is to make CLI output shorter. And relative paths actually are valid within the context of current shell.

Current dir's relative path can be also shown as `.` or `./` btw I think second is also readable but full path could be more in this exceptional case as not every starter user probably knows about it.